### PR TITLE
Do array exercises

### PR DESCRIPTION
**Pull request by CBinh710**
### Do array exercises
issue: [link](https://github.com/users/CBinh710/projects/8/views/1?pane=issue&itemId=80165780)

image

image

### CODE REVIEW

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

**Comment by CBinh710** [comment]
image

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+const item70 = randomNumbers.find(value => value === 70);
+
+// Using filter()
+const value30 = randomNumbers.filter(value => value === 30);
+
+const value70 = randomNumbers.filter(value => value === 70);
+
+// Using for()
+let found30 = false;
+
```
Comment (phuocduong-agilityio) [inline src/index.js]: Space between let/const block with another code. Not between let/const and let/const

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+let found70 = false;
+
+for (let i = 0; randomNumbers.length > i; i++) {
+
+  if (randomNumbers[i] === 30) {
```
Comment (phuocduong-agilityio) [inline src/index.js:56]: Find the item with value = 70 and 30 in randomNumbers? Do it match requirement?

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,89 @@
...
+//Using filter
+const numberCount = randomValues.filter((valueNumber) => typeof valueNumber === 'number').length;
+
+//Using for loop
+let count = 0;
+
+for (let i = 0; randomValues.length > i; i++) {
+
+  if (typeof randomValues[i] === 'number') {
+    count++
```
Comment (phuocduong-agilityio) [inline src/index.js]: Missing ;

File: src/index.js [hunk 4]
```diff
@@ -1,6 +1,89 @@
...
+
+const index70 = randomNumbers.indexOf(70);
+
+// Using find()
+const item30 = randomNumbers.find(value => value === 30);
+
+const item70 = randomNumbers.find(value => value === 70);
+
+// Using filter()
+const value30 = randomNumbers.filter(value => value === 30);
```
Comment (phuocduong-agilityio) [inline src/index.js]: Find the item with value = 70 and 30 in randomNumbers. Does it match requirement?

File: src/index.js [hunk 5]
```diff
@@ -1,6 +1,89 @@
...
+    maxNumber = randomNumbers[i];
+    maxIndex = i;
+  }
+
+}
+
+// 3. Plus each number in randomNumbers with 5. Result should be [15, 25, 55 …]. Find at least 2 ways to do this
+
+// Using Map()
+const plusFive = randomNumbers.map(num => num + 5);
```
Comment (phuocduong-agilityio) [inline src/index.js]: All your result didn't print? Why?

File: src/index.js [hunk 6]
```diff
@@ -1,6 +1,89 @@
...
+
 }
-exampleFunction(10, 20);
+
+// Exercise 2
+let randomNumbers = [10, 20, 50, 25, 70, 0, 55, 90];
+// 1. Find the item with value = 70 and 30 in randomNumbers. Find at least 4 ways to do this
+
+// Using indexOf()
+const index30 = randomNumbers.indexOf(30);
```
Comment (phuocduong-agilityio) [inline src/index.js]: Consider naming convention for all

File: src/index.js (hunk 1 above)
Comment (CBinh710) [reply src/index.js]: I understood and fixed that, thanks

File: src/index.js (hunk 2 above)
Comment (CBinh710) [reply src/index.js:56]: I changed all to fix with requirement, thanks

File: src/index.js (hunk 3 above)
Comment (CBinh710) [reply src/index.js]: i added, thanks

File: src/index.js (hunk 4 above)
Comment (CBinh710) [reply src/index.js]: I changed all to fix with requirement, thanks

File: src/index.js (hunk 5 above)
Comment (CBinh710) [reply src/index.js]: i had some mistake when test code so i removed all, i fixed that

File: src/index.js (hunk 6 above)
Comment (CBinh710) [reply src/index.js]: i renamed all

**Pull request merged**